test/evergreen.yml

```diff
@@ -3881,7 +3881,7 @@ tasks:
     commands:
       - func: "run data validation stress test checkpoint"
         vars:
-          run_test_checkpoint_args: -x -s 7 -C cache_size=1000MB,precise_checkpoint=true
+          run_test_checkpoint_args: -x -s 7 -C cache_size=1000MB,precise_checkpoint=true,preserve_prepared=true
```

Comment:
is there value in having seperate tasks for prepared oriented tests? It seems valuable to have non-prepared tests.

Note: test/checkpoint tests are scattered around a bit, there are a set of tests defined test/checkpoint/CMakeLists.txt as welll